cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -331,7 +448,10 @@ bool cros_gralloc_driver::get_resolved_format_and_use_flags(
 	struct combination *combo;
 
 	struct driver *drv = is_video_format(descriptor) ? drv_video_ : drv_render_;
-	if ((descriptor->use_flags & BO_USE_SCANOUT) && !(is_video_format(descriptor)))
+	if (drv_ivshmem_ && (descriptor->use_flags & BO_USE_SCANOUT) &&
```

Comment:
Policy of selecting driver getting complicated, would be better to introduce a new function for it.